librz/asm/arch/arc/gnu/arc-dis.c

```diff
@@ -86,8 +86,9 @@ typedef enum
 #define PUT_NEXT_WORD_IN(a)						\
   do									\
     {									\
-      if (is_limm == 1 && !NEXT_WORD (1))				\
+      if (is_limm == 1 && !NEXT_WORD (1)) {				\
```

Comment:
Please note, that this file is copied from binutils:
- https://github.com/rizinorg/rizin/issues/814
- https://github.com/rizinorg/rizin/issues/1084

Reply:
yes, i know.